package.json

```diff
@@ -10,13 +10,20 @@
     "type": "git",
     "url": "https://github.com/thought-machine/please-vscode"
   },
+  "bugs": {
+    "url": "https://github.com/thought-machine/please-vscode/issues"
+  },
   "publisher": "please-build",
-  "categories": [],
+  "categories": [
+    "Programming Languages",
+    "Debuggers"
+  ],
   "keywords": [],
   "engines": {
-    "vscode": "^1.36.0"
+    "vscode": "^1.52.0"
   },
   "activationEvents": [
+    "workspaceContains:.plzconfig",
```

Comment:
Is this an `and` or an `or`? Might want to activate even if there's no .plzconfig if we're opening a subfolder in a larger monorepo.

Reply:
It's an `or`